src/System.Text.Encoding.CodePages/src/System/Text/BaseCodePageEncoding.cs

```diff
@@ -340,5 +340,27 @@ internal unsafe void CheckMemorySection()
                 LoadManagedCodePage();
             }
         }
+
+        // Method assumes that memory pointer is aligned
+        internal static unsafe void ZeroMemAligned(byte* buffer, int count)
```

Comment:
Do we not have any API already that we can use to zero memory? This does not seem special in any way.

@GrabYourPitchforks ?

Reply:
we have Unsafe.InitBlockUnaligned and I am going to switch using it.